# goal
string filename_to_play #the fully qualified filename including file ending, example: "/home/user/sounds/hello.wav"
---
#result
bool success
---
#feedback
float32 position    # the current position in the media file as percentage between 0.0 and 1.0
int64 time          # the current time in the media file (in ms)
